pom.xml

```diff
@@ -442,7 +454,48 @@
             <version>${geronimo.jms.2.spec.version}</version>
             <!-- License: Apache 2.0 -->
          </dependency>
-
+         <dependency>
+            <groupId>jakarta.jms</groupId>
+            <artifactId>jakarta.jms-api</artifactId>
+            <version>${jakarta.jms-api.version}</version>
+            <!-- License: EPL 2.0 -->
+           <scope>provided</scope>
```

Comment:
Related to other comment, perhaps this scope definition should be removed?

Reply:
<dependency>
            <groupId>org.apache.geronimo.specs</groupId>
            <artifactId>geronimo-annotation_1.2_spec</artifactId>
            <version>${geronimo-annotation_1.2_spec.version}</version>
            <scope>provided</scope>
         </dependency> would be similar

Reply:
Perhaps, though I'd consider it somewhat seperate than the main-client-API case I was interested in as it pertains to the artemis-jakarta-client module. The comparible JMS spec entry to that case, right above it, does not have use a 'provided' scope (though it is expected to be compile scope...which is somewhat the point here).

I do personally typically omit scopes from dependencyManagement though and mainly just use it for versioning, I think its clearer to have the specific modules that want non-compile scopes indicate that themselves.